selftests/functional/test_basic.py

```diff
@@ -1,3 +1,4 @@
+# This Python file uses the following encoding: utf-8
```

Comment:
I'm not sure this was intentional (probably was), but it matches the regex `^[ \t\v]*#.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)"` given on PEP 263:

https://www.python.org/dev/peps/pep-0263/

Just in case it wasn't intentional, and you weren't aware of PEP263, you may choose to use a more standard notations:

```
   # coding=<encoding name>
```

But it's entirely up to you.  This one does the job nicely.